test/action_controller/serialization_test.rb

```diff
@@ -135,7 +135,7 @@ def render_fragment_changed_object_with_relationship
           like = Like.new(id: 1, likeable: comment, time: 3.days.ago)
 
           generate_cached_serializer(like)
-          like.likable = comment2
+          like.likeable = comment2
```

Comment:
well, caught it because it raised an error, undefined attribute.. makes me wonder if it was being used...